### DISCUSSION TITLE
In-Network file for plans with national networks.

### DISCUSSION THREAD

**Discussion by vijayanandramamoorthy** [CMSgov/price-transparency-guide#222] [General]
Our organization is a licensee of a national health plan association in which each licensee generally operates independent of the association and other licensees, offering insurance plans within a defined geographic region (e.g., a U.S. state or territory) and contracting directly with providers in that region, but all associated licensees’ agree to allow members to visit other licensees’ contracted providers and leverage their negotiated rates.  Although members rarely visit non-local providers, the current MRF requirements suggest that plans with this type of partnership would be required to report every licensees’ provider negotiated rates in their plan’s In-Network MRF file, meaning dozens of associated health plans would be expected to exchange, store, manage and publish the IN-file each month with ~500,000 redundant provider negotiated rates rather than each licensee publishing their own contracted rates (~10,000 for our plan) which would significantly reduce the cost, complexity, and usability of In-network MRF without impacting public accessibility of all relevant data.  

Can the MRF technical requirements be modified to solve for unnecessary data redundancy by allowing associated health plans to report their own contracted provider negotiated rates in that plan’s published MRF, then refer to associate plans’ MRF files that contain provider rates in other regions?   This could be accomplished by linking to other plans’ URLs in the location where the Local In-Network MRF file is published or adding a new data element to the MRF file itself, where we can list the URLs .  This recommendation would help consumers of the files as well, since it is not expected that the general public would have the infrastructure in place to download a consolidated in-network file containing ~500k providers (1TB+) and most users would not be interested in rates of non-local providers, but could easily choose to include that data at the users discretion.  

thank you

**Comment by ghost** [comment]
Great idea....file reference would greatly assist both the consumer and producer of the data.

**Comment by shaselton-usds** [comment]
@vijayanandramamoorthy @BobSyracuse 

Hopefully the [latest changes address this](https://github.com/CMSgov/price-transparency-guide/pull/342). Please let me know if you're still running into duplicative data redundancy RE: networks.
